.github/workflows/workflow.yml

```
@@ -197,3 +197,31 @@ jobs:
           echo "${{ secrets.FRANCE_API_DEPLOY_KEY }}" > /home/runner/.ssh/france_api_deploy
           chmod 600 /home/runner/.ssh/france_api_deploy
           ssh -i /home/runner/.ssh/france_api_deploy -o StrictHostKeyChecking=no deploy-api@fr.openfisca.org
+
+  check-api-version:
+    runs-on: ubuntu-latest
+    needs: [ deploy ]
+    outputs:
+      version-match: ${{ steps.check-api-version.outputs.version-match }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Cache build
+        id: restore-build
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ env.pythonLocation }}-build-${{ hashFiles('setup.py') }}
+      - name: Check api version
+        id: check-api-version
+        run: |
+          output=$(python "${GITHUB_WORKSPACE}/.github/check-api-version.py")
+          echo "::set-output name=version-match::$output"
+      - name: Check on failure
+        if: steps.check-api-version.outputs.version-match != 'True'
+        run: |
+          echo "The api package version and the pypi package version do not match"
```

It seems we are comparing against the _local_ version, not the latest one on Pypi, right? 🙂 

```suggestion
          echo "The version of the API deployed on fr.openfisca.org/legislation/latest does not match the local package version."
```